frontend/packages/ux-editor/src/components/Settings/SettingsUtils.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { type ReactElement } from 'react';
+import {
+  NavigationDataIcon,
+  QuestionmarkIcon,
+  NavigationSignIcon,
+  NavigationPayIcon,
+  NavigationReceiptIcon,
+} from '@studio/icons';
+import { PROTECTED_TASK_NAME_CUSTOM_RECEIPT } from 'app-shared/constants';
+
+export const taskNavigationType = (taskType: string) => {
+  if (taskType === 'receipt' || taskType === PROTECTED_TASK_NAME_CUSTOM_RECEIPT) {
+    return 'process_editor.configuration_panel_custom_receipt_accordion_header';
+  } else {
+    return `process_editor.task_type.${taskType}`;
+  }
+};
+
+export const taskNavigationIcon = (taskType: string, taskIcon?: string): ReactElement => {
+  switch (taskType) {
+    case 'receipt':
+    case PROTECTED_TASK_NAME_CUSTOM_RECEIPT:
+      return <NavigationReceiptIcon data-testid='receipt' className={taskIcon} />;
```

Comment:
It seems that these icons exist in aksel, so I'm wondering if we could use them directly:
```suggestion
      return <ReceiptIcon data-testid='receipt' className={taskIcon} />;
```

Reply:
They are not exactly the same. We want to use the same icons as they appear in the app's navigation, which includes a border and some other adjustments. Because of this, Anniken designed new icons, which we implement as SVGs in our project.